packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -2731,7 +2731,8 @@ const schema: SchemaType<"Users"> = {
   
   altAccountsDetected: resolverOnlyField({
     type: 'Boolean',
-    graphQLtype: 'Boolean!',
+    graphQLtype: 'Boolean',
+    nullable: true,
```

Comment:
I think this should be `nullable: false` combined with `...schemaDefaultValue(false)`, and backfilling any missing values as necessary. This was the direction we were trying to move in with the big non-nullability PR

Reply:
This is a resolver only field though? And I kind of feel like it's better to return `null` in this case when you don't have permission to see this data

Reply:
Oh I didn't notice that, ok seems reasonable 👍